package.json

```
@@ -7,8 +7,9 @@
     "dev": "vite dev",
     "build": "node scripts/pre-build.js && vite build",
     "preview": "vite preview",
-    "test": "npm run lint && npm run test:unit",
+    "test": "npm run lint && npm run check && npm run test:unit:coverage",
```

Unrelated, but figured we should include `npm run check` in our default `test` command.

---

I'm ok with adding `npm run check`, but I don't think we should make this run `test:unit:coverage` instead of `test:unit` because `test:unit:coverage` disables the default behavior of watching the tests and source and rerunning tests on change. It also makes the tests take longer.

We can add a `coverage` script that is just an alias for `test:unit:coverage` if we want it to be easier to generate the coverage report.

```suggestion
    "test": "npm run lint && npm run check && npm run test:unit",
```

---

Right, I forgot that I added `run` to the command to ensure we weren't rerunning coverage with the watch mode (since the coverage would only update based on the tests that were run, which was misleading). I'll rever that change.